packages/UI/src/components/organisms/cards/PositionCard/PositionCard.tsx

```diff
@@ -0,0 +1,177 @@
+import { useState } from 'react';
+import Box from '../../../atoms/Box';
+import Button from '../../../atoms/Button';
+import Typography from '../../../atoms/Typography';
+import LikeButton from '../../../molecules/InteractionButtons/LikeButton';
+import SaveButton from '../../../molecules/InteractionButtons/SaveButton';
+import ShareButton from '../../../molecules/InteractionButtons/ShareButton';
+import {
+  TagsContainer,
+  CardDescription,
+  ExpandableBlurb,
+} from '../StyledCommonComponents';
+
+import {
+  Container,
+  Header,
+  Thumbnail,
+  Actions,
+  Details,
+} from './Styled.PositionCard';
+import type { PositionCardProps } from '.';
+
+const PositionCard = ({
+  interaction,
+  status,
+  buttonStyle,
+  minimal,
+  content,
+  handleMoreDetails,
+  handleApply,
+}: PositionCardProps) => {
+  const [expanded, setExpanded] = useState(status);
+
+  const isMobile = window.innerWidth < 640;
+
+  const MinimalPositionCard = () => {
+    return (
+      <Container minimal>
+        <Thumbnail minimal>
+          <Box flexDirection="column" gap="8px" width="max-content">
+            <Typography type="h3">{content.title}</Typography>
+            <Typography type="h5">{content.level}</Typography>
+            <hr />
+          </Box>
+          <Box flexDirection="column" gap="8px" width="fit-content">
+            <Typography type="h5">time committment</Typography>
+            <Typography type="pSmall">{content.timeCommittment}</Typography>
+          </Box>
+        </Thumbnail>
+        <Actions>
+          <Box width="100%" padding="0px 24px" justifyContent="flex-end">
+            <Box gap="16px">
+              <Button
+                onClick={handleMoreDetails}
+                buttonSize="standard"
+                buttonType={buttonStyle === 'b' ? 'secondary' : 'alternative'}
+              >
+                more details
+              </Button>
+              <Button
+                onClick={handleApply}
+                buttonSize="standard"
+                buttonType={buttonStyle === 'b' ? 'primary' : 'alternative'}
+              >
+                apply
+              </Button>
+            </Box>
+          </Box>
+        </Actions>
+      </Container>
+    );
+  };
```

Comment:
@Nouri-Anouar   I just checked your PR really quick and I noticed you have nested components definitions (`PositionCard` has another component definition nested inside called `MinimalPositionCard`).
Check out this video on why it's discouraged to do such a thing and it's recommended to extract them outside
https://www.youtube.com/watch?v=2sAdzy90GtE